src/bundle/Resources/public/scss/_filters.scss

```diff
@@ -0,0 +1,37 @@
+.ez-filters {
+    display: flex;
+    justify-content: space-between;
+    border-top: 1px solid $ez-color-secondary;
+    transform: scaleY(1);
+    transform-origin: top center;
+    transition: all 0.2s $ez-admin-transition;
+    flex-wrap: wrap;
+
+    &--collapsed {
```

Comment:
Maybe you should extract it to some kind of mixin? Like `collapsible`. Thanks to that, it'll be easier to reuse the same style of collapsible state in any other components.